.taskcluster.yml

```diff
@@ -166,3 +167,55 @@ tasks:
                       --
                       --channel=${browser.channel}
                       ${operation.extra_args};
+    - $if: tasks_for == "github-pull-request"
+      # PR tasks that aren't running the changed tests
+      then:
+        $if: event.action in ['opened', 'reopened', 'synchronize']
+        then:
+          $map:
+             - name: lint
+               description: >-
```

Comment:
Where does this description end up?

Reply:
Real question: can we have this as the name in checks? I didn't know there was a separate description before I saw this.